pkg/identity/cache/allocator.go

```diff
@@ -217,6 +229,14 @@ func AllocateIdentity(ctx context.Context, lbls labels.Labels) (*identity.Identi
 // identity again. This function may result in kvstore operations.
 // After the last user has released the ID, the returned lastUse value is true.
 func Release(ctx context.Context, id *identity.Identity) (bool, error) {
+	lastUse, err := release(ctx, id)
```

Comment:
One question - what is the scope of `lastUse` - is it global or local to this node?

Reply:
Ah, I believe that this is the last local reference to this identity, but I seem to recall @jrajahalme talking about this not having the semantics that one might expect. This part may end up going away when I rebase against #7539 .